compiler/src/dotty/tools/dotc/typer/Typer.scala

```diff
@@ -1947,7 +1950,7 @@ class Typer extends Namer
   }
 
   protected def makeImplicitFunction(tree: untpd.Tree, pt: Type)(implicit ctx: Context): Tree = {
```

Comment:
Sadly the here the `AppliedType` arrives.
It gets deconstructed by `defn.Function` which simply parses the class of the type constructor. :disappointed:

Reply:
Should `localQualifier` be one of the arguments of `LocalFunctionN`?